pkg/util/kubectl/kubectl.go

```diff
@@ -33,7 +33,7 @@ func BinaryForClusterVersion(version *semver.Version) (string, error) {
 	case 20:
 		binary = "kubectl-1.20"
 	case 21:
-		binary = "kubectl-1.22"
```

Comment:
Why this change?

Reply:
Why'd you want to run the binary for `1.22` if the minor version is `1.21`? I'd like to understand the need behind that.

I was getting errors when running seed-controller-manager locally; unable to apply addon, file(binary) not found.

Reply:
cc @xmudrii

Reply:
My idea was to not create another PR for such a simple change. But anyway, I get your point now.

Reply:
Updated